build/patches/Disable-all-predictors-code.patch

```diff
@@ -238,15 +238,15 @@ diff --git a/components/optimization_guide/features.gni b/components/optimizatio
 diff --git a/components/permissions/features.cc b/components/permissions/features.cc
 --- a/components/permissions/features.cc
 +++ b/components/permissions/features.cc
-@@ -65,7 +65,7 @@ const base::Feature kPermissionPredictionServiceUseUrlOverride{
- 
- const base::Feature kPermissionOnDeviceNotificationPredictions{
-     "PermissionOnDeviceNotificationPredictions",
+@@ -61,7 +61,7 @@ const base::Feature kPermissionChipRequestTypeSensitive{
+ // for the Web Permission Predictions Service.
+ const base::Feature kPermissionPredictionServiceUseUrlOverride{
+     "kPermissionPredictionServiceUseUrlOverride",
```

Comment:
the flag to disable is `PermissionOnDeviceNotificationPredictions` not `kPermissionPredictionServiceUseUrlOverride`

Reply:
This is a guarded flag, not a disabled one. The guard comment should have included a change to the above line.

Or you're saying that the guarded flag `PermissionOnDeviceNotificationPredictions` went from disabled by default to enabled and the fuzzy patching picked the wrong line?

Reply:
I am saying that you missed the `PermissionOnDeviceNotificationPredictions` flag, which has to be checked, but I saw that now you added it.
in my opinion checking `kPermissionPredictionServiceUseUrlOverride` one is useless, because it is only used in tests

Reply:
> in my opinion checking kPermissionPredictionServiceUseUrlOverride one is useless, because it is only used in tests

I did not introduce this; so either it has always been there or fuzz-applying moved it (because it was not anchored to the correct context).